### ISSUE TITLE
Problems with TEAMEngine based tests

### ISSUE THREAD

**Issue by hwbllmnn** [INSPIRE-MIF/helpdesk-validator#917]
I'm trying to get the validator v2023.0 running with the TEAMEngine tests. According to the release notes I need to add these lines to the `etf-config.properties` file:

```
etf.testdrivers.teamengine.url = http://cite.opengeospatial.org/teamengine
etf.testdrivers.teamengine.username = <username>
etf.testdrivers.teamengine.password = <deleted>
```

I tried to add these to the `etf-config.properties` within the `validator.war` (inside `WEB-INF/classes`) and also tried adding them to the `etf-config.properties` in the `/etf/config` folder. However, I always get these errors in the log:

```
2023-01-31 10:22:01.154 [main] INFO  d.i.e.t.ComponentContainer - Preparing Test Driver TEAM Engine test driver OGC API Features 1.0.1-SNAPSHOT
2023-01-31 10:22:01.829 [main] ERROR d.i.e.t.TestDriverLoader - Failed to load component
de.interactive_instruments.etf.component.ComponentLoadingException: Loading of component JAR "etf-tetd-api-features-1.0.1-SNAPSHOT.jar" failed  Could not retrieve Executable Test Suites with TEAM Engine application web interface  HTTP error fetching URL
	at de.interactive_instruments.etf.testdriver.ComponentContainer.loadAndInit(ComponentContainer.java:141)
	at de.interactive_instruments.etf.testdriver.TestDriverLoader.load(TestDriverLoader.java:136)
	at de.interactive_instruments.etf.testdriver.TestDriverLoader.load(TestDriverLoader.java:149)
	at de.interactive_instruments.etf.testdriver.DefaultTestDriverManager.loadAll(DefaultTestDriverManager.java:71)
	at de.interactive_instruments.etf.webapp.controller.TestDriverController.init(TestDriverController.java:120)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	...
```

Consequently, the OGC API Features and the Direct WFS tests are not available in the validator. What am I doing wrong? I also tried to switch the URL to the https variant to no effect. I can log into the online teamengine account just fine with the configured credentials.

**Comment by jbarreroj** [comment]
Dear @hwbllmnn,

We have tried different approaches to reproduce the error without any satisfactory outcome. We would kindly like to ask you for further details that might help us recreate it.

Anyway, we will provide you with some points to consider that may help you solve the problem. 

- First, stop the Docker image, delete the etf folder (the one that contains the config and logs folders) and restart it, or create it again, the one you prefer.
- Another possible workaround might be to use the docker image specified in the release: _docker run --name inspire-validator -d -p 8090:8090 -v ~/etf:/etf docker.pkg.github.com/inspire-mif/helpdesk-validator/inspire-validator:2023.0_ 
As a note, we recommend removing the etf folder too in this second case.

Please let us know if the issue has not been solved with these tips.

Kind regards.

**Comment by hwbllmnn** [comment]
Hi @jbarreroj,

your answer seems to indicate that I need to add the credentials to the config in the etf config folder, is that correct? According to the release notes the config should be changed in the `.war` file directly, I'm not sure how I would do that when running the official docker image directly. Of course I could manually edit the `.war` inside the container and then restart it, however, this would have to be done every time the container is recreated.

Best regards

**Comment by jbarreroj** [comment]
Dear @hwbllmnn, 

in the release is stated to modify the files in the .war because it is used to create a folder named _etf_ within the user's folder. If it is already created, any changes made to the war will not be in that folder that is used to know your credentials. This is why we recommended removing the _etf_ folder as the first option, as your error may be due to the existence of an earlier version of the _etf_ folder.

In the second possible solution we gave you, we recommended using docker. In this case, you wouldn't need to change the .war, you only need to add your credentials in etf/config/etf-config.properties (being _etf_ the folder we mentioned before). Once you do it, it doesn't matter if you restart it because it will have the previous file and it won't modify it.

Kind regards.
